internal/worker/changestream/doc.go

```diff
@@ -1,11 +1,17 @@
-// Copyright 2025 Canonical Ltd. Licensed under the AGPLv3, see LICENCE file for
-// details.
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
```

Comment:
I have no idea why there is that empty line in between the copyright and the licence here. In my editor it doesn't show.